src/Cafe/HW/Espresso/Debugger/DebugSymbolStorage.h

```diff
@@ -45,12 +45,17 @@ class DebugSymbolStorage
 
 	static void ClearRange(MPTR address, uint32 length)
 	{
+		if (length == 0)
+			return;
 		s_lock.lock();
-		while (length > 0)
+		for (;;)
 		{
 			auto itr = s_typeStorage.find(address);
 			if (itr != s_typeStorage.end())
 				s_typeStorage.erase(itr);
+			// break if length has surpassed zero and wrapped around
+			if (length - (uint32)4u > length)
```

Comment:
either `(uint32)4` or `4u` is enough, you dont need both.

But also why not just do `if (length < 4)`

Reply:
> you dont need both.

I knew that but I was thinking "an unsigned integer is not guaranteed to be 32-bit"
> But also why not just do `if (length < 4)`

You make a good point...